package.json

```diff
@@ -69,7 +68,7 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "yarn sfdx-husky-commit-msg",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
Updated to use the commitlint command being called by `sfdx-husky-commit-msg` that is part of the dev-scripts project. Removed the import since we are no longer calling the script.

Reply:
I think we're missing adding a dependency since I'm getting `/bin/sh: commitlint: command not found` error when running a test commit on this branch.

Reply:
Thanks for the catch, I didn't clean properly before testing it again